JavaAssignment/src/ankitaG/Program5.java

```diff
@@ -0,0 +1,24 @@
+package ankitaG;
+
+public class Program5 {
+	float principalAmt;
+	float rate;
+	int year;
+	float interest;
+
+	void simpleInterestCal(float p, float r, int y) {
+		principalAmt = p;
+		rate = r;
```

Comment:
we are not using this variable out of this method right, so why have you created this variable as an instance variable. Please make it local variable.